Components/LiveStream.tsx

```diff
@@ -109,39 +113,28 @@ export const LiveStream: React.FC<LiveStreamProps> = ({
       setIsHost(false);
       showMessage('You left the channel');
       // sendDataToBackend();
-      navigation.navigate('Home');
+      navigation.navigate('Home' as keyof RootStackParams);
     } catch (e) {
       console.log(e);
     }
   };
 
   return (
     <View style={styles.main}>
-      <View style={styles.mapContainer}>
-        {isJoined && (
-          <LiveMap
-            setTripCoordinates={setTripCoordinates}
-            setTimestamps={setTimestamps}
-          />
-        )}
-      </View>
       <View style={[styles.btnContainer, {width: isHost ? '40%' : '30%'}]}>
         <CustomButton
-          onPress={join}
           name={isHost ? 'End Stream' : 'Leave'}
           onPress={leave}
         />
       </View>
-      <View
-        style={styles.scroll}
-        contentContainerStyle={styles.scrollContainer}>
```

Comment:
Looks like you cleaned up the code to separate the map component with the steamer component, making them more modular. Nice job!